### PR TITLE
Debt auction bid size

### PR DESCRIPTION
**Pull request by astronautthis**
@Gala-G

### CODE REVIEW

**Comment by Gala-G** [comment]
@patrick-j-govalpha 

Could you please review this doc before I merge it?
It looks good to me, from a non-technical eye.

thank you so much
g

File: parameter-index/debt-auction/param-bid-size.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+# Debt Auction Bid Size
+
+```
```
Comment (patrick-j-govalpha) [inline parameter-index/debt-auction/param-bid-size.md:4]: Don't need the "```"

File: parameter-index/debt-auction/param-bid-size.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+>**Alias:  
+>**Parameter Name: `sump`  
+>**Containing Contract: Vow  
+>**Scope: System  
+>**Technical Docs: https://docs.makerdao.com/smart-contract-modules/system-stabilizer-module/vow-detailed-documentation
+```
```
Comment (patrick-j-govalpha) [inline parameter-index/debt-auction/param-bid-size.md:10]: Don't need the "```"

File: parameter-index/debt-auction/param-bid-size.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (patrick-j-govalpha) [inline parameter-index/debt-auction/param-bid-size.md:1]: Don't need this

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]
Content looks fine. A couple of formatting issues to sort out

**Comment by astronautthis** [comment]
> @patrick-j-govalpha
> 
> Could you please review this doc before I merge it? It looks good to me, from a non-technical eye.
> 
> thank you so much g

Thanks Gala! I've fixed the formatting issues @patrick-j-govalpha pointed out

**Pull request merged**